server/ecs/app.py

```diff
@@ -59,10 +59,15 @@ def _before_adding_routes(app, app_config):
 
         PLAUSIBLE_URL = "https://plausible.io"
 
+        HUBSPOT_JS_URL = "https://js.hsforms.net"
+
+        HUBSPOT_FORMS_URL = "https://forms.hsforms.com"
+
         csp = {
-            "default-src": ["'self'"],
-            "connect-src": ["'self'", PLAUSIBLE_URL] + extra_connect_src,
-            "script-src": ["'self'", "'unsafe-eval'", PLAUSIBLE_URL] + script_hashes,
+            "default-src": ["'self'", HUBSPOT_FORMS_URL],
```

Comment:
Seems like if we have the form URL in default src, the URL won't need to be in other directives 😄?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src